### PR TITLE
feat(cloud-web): fix sidebar for smaller screens

### PR DESCRIPTION
**Pull request by varun257**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #DTRSD-10444
| License          | BSD 3-Clause

## Description

remove classes to remove overridden styles

Signed-off-by: varun257 <varun257@gmail.com>

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/index.html [hunk 1]
```diff
@@ -72,9 +72,7 @@
                         data-sidebar-namespace="'account'"
                     ></div>
 
-                    <div
-                        class="col-md-9 col-md-offset-3 col-lg-10 col-lg-offset-2 px-0"
-                    >
+                    <div class="col-lg-10 col-lg-offset-2 px-0">
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/client/app/index.html:75]: You sure this works ? Did you try every screen resolution ? This issue is present a lot in the Managers

Comment (varun257) [reply packages/manager/apps/dedicated/client/app/index.html:75]: Yes it looks good in all resolutions. col-md-9 is overwritten in main css with width for media queries

**Comment by antleblanc** [comment]
@varun257 Is this Pull Request still needed?
Ticket `DTRSD-10444` is already marked as closed.

Thank you!

**Comment by antleblanc** [comment]
@varun257 I close this Pull Request due to inactivity.
Could you please also remove the branch?

Please do not hesitate to re-open it if needed.

Thank you!

**Pull request closed without merging**